Could not locate the assembly "Microsoft.Python.Analysis.Engine"

Saw this in a build log:

```
warning MSB3245: Could not resolve this reference. Could not locate the assembly "Microsoft.Python.Analysis.Engine". Check to make sure the assembly exists on disk. If this reference is required by your code, you may get compilation errors. [E:\A\_work\8\s\src\LanguageServer\Impl\Microsoft.Python.LanguageServer.csproj]
```

Just a warning, but we should clean up the csproj to not refer to the engine we haven't used in 8 months 😄

--- a/src/LanguageServer/Impl/Microsoft.Python.LanguageServer.csproj
+++ b/src/LanguageServer/Impl/Microsoft.Python.LanguageServer.csproj
@@ -33,14 +33,6 @@
         <PackageReference Include="NewtonSoft.Json" Version="12.0.2" />
         <PackageReference Include="StreamJsonRpc" Version="2.1.55" />
     </ItemGroup>
-    <ItemGroup Condition="$(AnalysisReference) != ''">
-        <Reference Include="Microsoft.Python.Analysis.Engine">
-            <HintPath>$(AnalysisReference)/Microsoft.Python.Analysis.Engine.dll</HintPath>
-        </Reference>
-        <Content Include="$(AnalysisReference)/**/*">
-            <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-        </Content>
-    </ItemGroup>
     <ItemGroup>
       <ProjectReference Include="..\..\Analysis\Ast\Impl\Microsoft.Python.Analysis.csproj" />
       <ProjectReference Include="..\..\Caching\Impl\Microsoft.Python.Analysis.Caching.csproj" />

